include/mbedtls/bignum.h

```diff
@@ -106,33 +106,47 @@
  * 32-bit integers can be forced on 64-bit arches (eg. for testing purposes)
  * by defining MBEDTLS_HAVE_INT32 and undefining MBEDTLS_HAVE_ASM
```

Comment:
I think the comment needs to be expanded to say you can dictate your word size by specifying `MBEDTLS_HAVE_INT32`, `MBEDTLS_HAVE_INT64` or `MBEDTLS_HAVE_UDBL` - (but see other comment about how to define `mbedtls_t_udbl`).